src/pages/AnimeFenix/main.ts

```diff
@@ -0,0 +1,61 @@
+import { pageInterface } from '../pageInterface';
+
+export const AnimeFenix: pageInterface = {
+  name: 'AnimeFenix',
+  domain: 'https://animefenix.tv',
+  languages: ['Spanish'],
+  type: 'anime',
+  isSyncPage: url => {
+    const path = url.split('/')[3];
+    return path === 'ver';
+  },
+  sync: {
+    getTitle: url => {
+      const titleElement = document.querySelector('.hero h1');
+      let title = titleElement?.textContent ? titleElement.textContent.trim() : null;
+
+      if (!title) {
+        const urlTitle = url.split('/')[4];
+        title = urlTitle ? urlTitle.replace(/-/g, ' ').replace(/\b\w/g, l => l.toUpperCase()) : '';
+      }
```

Comment:
This section will still return a wrong title.
![image](https://github.com/MALSync/MALSync/assets/12599334/a3da1f27-dfab-4ac6-92e3-e80cfd952d25)